app/src/main/java/turtler/voyageur/activities/BaseActivity.java

```diff
@@ -268,6 +327,55 @@ public void onError(int i, Exception e) {
         }
     }
 
+    public void saveImageToParse(Location mLastLocation, File resizedFile) {
+        String lat = Double.toString(mLastLocation.getLatitude());
+        String lon = Double.toString(mLastLocation.getLongitude());
+        ParseObject parseImage = ParseObject.create("Image");
```

Comment:
Can you fix the Image model class, and create getters and setters, constructor if needed. We should be able to do:
Image image = new Image();
image.setLatitude(lat);
etc